spec/mailers/event_confirmation_spec.rb

```diff
@@ -15,7 +15,9 @@
     end
 
     it 'renders the body' do
-      expect(mail.body.encoded).to(have_content('Thanks for volunteering'))
+      expect(mail.body.encoded).to(have_content('Your sign-up for Event 1 on 12/31/2029 has been confirmed!'));
+      expect(mail.body.encoded).to(have_content('The event will take place at Event Location at 05:30 pm.'));
+      expect(mail.body.encoded).to(have_content('You will receive 1.0 service hours for participating.'));
     end
   end
 end
```

Comment:
Nice job adding security tests to this feature. If I remember correctly, Natalia wanted a message on the email saying something like "Check the webpage for more info about an event" or adding a line saying to contact the officer in charge.

Reply:
Just met with Natalia, including something like "See the more details about the event in the Breaking Barriers website" should be sufficient

Reply:
Yep. Was planning on doing it later, but I'll just add to this PR.